apps/tailwind-components/app/components/button/FilterWellContainer.vue

```diff
@@ -1,41 +1,68 @@
 <script lang="ts" setup>
-import { nextTick, onBeforeUnmount, onMounted, ref, watch } from "vue";
-import Button from "../Button.vue";
+import { ref, useTemplateRef, watch, onMounted, onBeforeUnmount } from "vue";
+import ButtonText from "../button/Text.vue";
 
-const props = defineProps<{
+defineProps<{
   id: string;
-  size: number;
 }>();
+
+const buttonContainer = useTemplateRef<HTMLDivElement>("filterButtonContainer");
```

Comment:
why use a TemplateRef instead of an event for model value ?

Reply:
removed